lib/quick_spreadsheet.rb

```diff
@@ -3,50 +3,92 @@
 
 class QuickSpreadsheet
   def self.args
-    puts "title: string"
-    puts "header_row: Array<string>"
-    puts "body_rows: Array<Array<string>>"
+    puts "filename: string (optional)"
     puts "folder_path: string (optional)"
+    puts "Variant 1:"
+    puts "  title: string"
+    puts "  header_row: Array<string>"
+    puts "  body_rows: Array<Array<string>>"
+    puts "Variant 2:"
+    puts "  sheets: Array<Array<"
+    puts "    title: string"
+    puts "    header_row: Array<string>"
+    puts "    body_rows: Array<Array<string>>"
   end
 
   def self.call(args)
-    title       = args[:title]
-    header_row  = args[:header_row]
-    body_rows   = args[:body_rows]
-    folder_path = args[:folder_path] || "#{Dir.pwd}/tmp"
-    format      = :xls
+    time = Time.now
+    file_timestamp = time.strftime("%Y-%m-%d-%H.%M")
 
+    folder_path = args[:folder_path] || "#{Dir.pwd}/tmp"
     FileUtils::mkdir_p(folder_path) unless File.exists?(folder_path)
+    format      = :xls
 
-    time = Time.now
-    file_timestamp = time.strftime("%Y-%m-%d-%H.%M")
-    filename = "#{title.gsub(" ", "")}_g#{file_timestamp}.#{format.to_s}"
+    sheets = if args[:sheets]
+      args[:sheets]
+    else
+      [{
+        title:      args[:title],
+        header_row: args[:header_row],
+        body_rows:  args[:body_rows]
+      }]
+    end
```

Comment:
imo this is easier to read if you align the indentation for the conditional keywords:

```
sheets = 
  if args[:sheets]
    args[:sheets]
  else
    [{
      title:      args[:title],
      header_row: args[:header_row],
      body_rows:  args[:body_rows]
    }]
end
```